apps/minifront/src/components/ibc/layout.tsx

```diff
@@ -13,7 +13,7 @@ export const IbcLayout = () => {
             direction='right'
             // Negative calculated margin giving lint issue
             /* eslint-disable-next-line tailwindcss/enforces-negative-arbitrary-values */
-            className='invisible absolute -top-44 right-0 z-0 -mr-[calc(30vw-3px)] size-[30vw] text-stone-300 md:visible'
+            className='invisible absolute -top-32 right-0 z-0 -mr-80 size-80 text-stone-300 md:visible'
```

Comment:
The resizing of these arrows are causing strange behaviors. Going to make them fixed sizes.